kubelet-log-config/kubelet-log-config.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kubelet-log-config
+  namespace: default
+  labels:
+    k8s-app: kubelet-log-config
+spec:
+  selector:
+    matchLabels:
+      name: kubelet-log-config
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        name: kubelet-log-config
+    spec:
+      nodeSelector:
+        k8s-app: kubelet-log-config
+      hostPID: true
+      containers:
+        - name: startup-script
+          image: gcr.io/google-containers/startup-script:v1
+          imagePullPolicy: Always
+          securityContext:
+            privileged: true
+          env:
+          - name: STARTUP_SCRIPT
+            value: |
+              set -o errexit
+              set -o pipefail
+              set -o nounset
+
+              echo "Update kubelet config file with customized log max size and files."
+              echo "containerLogMaxSize: \"100Mi\"" >> /home/kubernetes/kubelet-config.yaml
```

Comment:
Add a logic to check if the file already has this setting. Having the same setting twice may break things and it may happen if the pod got restarted for some reason